### PR TITLE
Bug 1650722 - Publish What's New panel message for 80

### PR DESCRIPTION
**Pull request by piatra**
And archived older 76 messages.
Still waiting on the CTA link for the message.

### CODE REVIEW

File: whats-new-panel.yaml [hunk 1]
```diff
@@ -113,22 +35,59 @@
...
   trigger:
     id: whatsNewPanelOpened
-- id: WHATS_NEW_BADGE_78
+- id: WHATS_NEW_BADGE_80
   content:
     action:
       id: show-whatsnew-button
     badgeDescription:
       string_id: cfr-badge-reader-label-newfeature
-    bucket_id: WHATS_NEW_BADGE_78
```
Comment (ncloudioj) [inline whats-new-panel.yaml:122]: Can you also copy `WHATS_NEW_BADGE_78` to `archive/whats-new-panel-archived.yaml`?

File: whats-new-panel.yaml [hunk 2]
```diff
@@ -113,22 +35,59 @@
...
+    id: whatsNewPanelOpened
+- id: WHATS_NEW_COOKIES_PROTECTION_80
+  template: whatsnew_panel_message
+  order: 1
+  content:
+    bucket_id: WHATS_NEW_COOKIES_PROTECTION_80
+    published_date: 1598313600000
+    title:
+      string_id: cfr-whatsnew-clear-cookies-header
+    icon_url: chrome://browser/content/logos/tracking-protection.svg
```
Comment (ncloudioj) [inline whats-new-panel.yaml:82]: Just to confirm, it says to use `illustration-cookie-blocked-32.svg` in the doc specs. Did we still want to use the tracking protection one?

File: whats-new-panel.yaml [hunk 3]
```diff
@@ -113,22 +35,59 @@
...
+    icon_alt:
+      string_id: cfr-whatsnew-clear-cookies-image-alt
+    body:
+      string_id: cfr-whatsnew-clear-cookies-body
+    cta_url: "https://blog.mozilla.org/blog/2020/08/04/latest-firefox-rolls-out-enhanced-tracking-protection-2-0-blocking-redirect-trackers-by-default/"
```
Comment (ncloudioj) [inline whats-new-panel.yaml:87]: Looks like this link is en-US only, also the CTA link was specified as some other link. Can you confirm?

File: whats-new-panel.yaml [hunk 4]
```diff
@@ -113,22 +35,59 @@
...
     target: whats-new-menu-button
   frequency:
     lifetime: 100
   priority: 5
-  targeting: firefoxVersion == 78 && (usesFirefoxSync || hasAccessedFxAPanel || currentDate|date
-    - 2419200000 >= profileAgeCreated) && (!messageImpressions['WHATS_NEW_BADGE_78']
-    || (messageImpressions['WHATS_NEW_BADGE_78']|length >= 1 && currentDate|date -
-    messageImpressions['WHATS_NEW_BADGE_78'][0] <= 345600000)) && 'browser.messaging-system.whatsNewPanel.enabled'|preferenceValue
+  targeting: firefoxVersion == 80 && (usesFirefoxSync || hasAccessedFxAPanel || currentDate|date
+    - 2419200000 >= profileAgeCreated) && (!messageImpressions['WHATS_NEW_BADGE_80']
```
Comment (ncloudioj) [inline whats-new-panel.yaml:51]: Hmm, I am not sure if those filters `(usesFirefoxSync || hasAccessedFxAPanel || currentDate|date
    - 2419200000 >= profileAgeCreated)` are still relevant to 80, could you confirm?

File: whats-new-panel.yaml (hunk 3 above)
Comment (piatra) [reply whats-new-panel.yaml:87]: I'm waiting back on this.

File: whats-new-panel.yaml (hunk 2 above)
Comment (piatra) [reply whats-new-panel.yaml:82]: This is a mistake.

File: whats-new-panel.yaml (hunk 4 above)
Comment (piatra) [reply whats-new-panel.yaml:53]: Added comment.

File: whats-new-panel.yaml (hunk 3 above)
Comment (piatra) [reply whats-new-panel.yaml]: Publishing en-* only

**Pull request merged**